web/libs/editor/src/components/Comments/Comment/LinkState.scss

```diff
@@ -88,6 +88,10 @@
     flex: 1;
     color: var(--text-color, var(--sand_900));
     min-width: 0;
+    //// @todo use these styles if we don't need multiline texts
```

Comment:
Instead of commenting out, would it make sense to just add a modifier `truncate` which we could later use, even possibly conditionally?

Reply:
I'm always for reducing todo's and commented out code if possible.

Reply:
I agree in general, but adding a property/modifier/branch which is not used is actually less visible and won't allow to easily find redundant code. I'll think what to do here and will ask others about design, so maybe I'll just left one final solution.